ext/.gitignore

```diff
@@ -0,0 +1 @@
+/supl/
```

Comment:
Not sure if we should introduce a new local `.gitignore`, or use the one from the root directory, @SebastianBoe?

Reply:
I'd prefer to use the root directory, that makes it easier to get a global overview of ignored directories.

Reply:
Should have checked, wasn't aware of this file. I was also thinking that I should maybe also include libsupl.zip which is the name of the file you download.